pkgs/misc/drivers/epson-tm-m30ii/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchurl, cups, cmake }:
```

Personally I recommend:

```suggestion
{ lib
, stdenv
, fetchurl
, cups
, cmake
}:
```